fiftyone/server/view.py

```diff
@@ -213,19 +209,15 @@ def get_dataset_view(
         a :class:`fiftyone.core.view.DatasetView`
     """
     dataset = fod.load_dataset(dataset_name)
+    dataset.reload()
 
-    # Load a previously saved view
-    if view_name is not None and dataset.has_saved_view(view_name):
-        view = dataset.load_saved_view(view_name)
-        view.reload()
-        return
```

Comment:
@kaixi-wang apparently this code path wasn't ever being used? It wasn't actually returning the view...

Reply:
oh good catch! this used to and is supposed to return the saved view